data/sql/updates/pending_db_world/2024_09_01_00.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 17803;
+
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 17803) AND (`source_type` = 0) AND (`id` IN (0));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
```

Comment:
@Carriola-wotlk I am very curious what was the issue ?
I still see the chance field at 100%.

Reply:
castFlag set to 32
![immagine](https://github.com/user-attachments/assets/db2c8cd4-26d9-48e6-85cd-e5cc94c40064)

However, I just realized that I only fixed it for Normal dungeons. I'll update the query to also include Heroic dungeons